components/tools/OmeroPy/src/omero/testlib/__init__.py

```diff
@@ -1440,10 +1304,95 @@ def fullImport(self, client):
         finally:
             proc.close()
 
+    # Assert methods
     def assertImport(self, client, proc, folder):
         hashes = self.upload_folder(proc, folder)
         handle = proc.verifyUpload(hashes)
         cb = CmdCallbackI(client, handle)
         rsp = self.assertPasses(cb)
         assert 1 == len(rsp.pixels)
         return rsp
+
+    def assertWrite(self, mrepo2, filename, ofile):
```

Comment:
as we already started to follow pep standards why these doesn't follow naming convention?